data_registry/views.py

```diff
@@ -200,6 +200,29 @@ def download_export(request, id):
     )
 
 
+def datasets_metadata(request):
+    return JsonResponse(
+        list(
```

Comment:
I assume `list()` is necessary, and Django doesn't format the query result properly without it?

Reply:
Without it I get `Object of type MultilingualCollectionQuerySet is not JSON serializable`